### PR TITLE
Cbpay

### PR DESCRIPTION
**Pull request by cfluke-cb**
Modify the buy page to list options on onramp providers
add coinbase as an option

I am not sure what descriptions or details you may want to add each onramp provider. I want to remain neutral on that piece.

Wallet Not Connected
Screenshot 2022-12-02 at 12 59 14 PM

Wallet Connected
Screenshot 2022-12-02 at 12 59 23 PM

### CODE REVIEW

File: src/pages/Beta/Buy/index.tsx [hunk 1]
```diff
@@ -11,16 +21,62 @@ export default function BuyV2() {
...
+    )
+  } else {
+    return (
+      <TopContainer>
+        <Card>
+          <CardTitle>
+            <Title>Coinbase Pay</Title>
+          </CardTitle>
+          <PayButton variant="primary" onClick={() => buy('coinbase')} isDisabled={!account}>
+            <CBIcon src="./images/coinbase_coin_pay_blue.svg" />
```
Comment (SarjuHansaliya) [inline src/pages/Beta/Buy/index.tsx:64]: can you please move this image to `src/assets/svg` folder ?

**Review by SarjuHansaliya — CHANGES_REQUESTED** [review]
Thanks for the changes. I have added very small comments and also can you please also update PR with screenshot on how page looks now ?

File: src/constants/index.ts [hunk 2]
```diff
@@ -85,6 +85,7 @@ export const BIG_INT_SECONDS_IN_WEEK = JSBI.BigInt(60 * 60 * 24 * 7)
 export const MIN_ETH: JSBI = JSBI.exponentiate(JSBI.BigInt(10), JSBI.BigInt(16)) // .01 ETH
 
 export const MOONPAY_PK = process.env.REACT_APP_MOONPAY_PK || ''
+export const COINBASE_PK = process.env.REACT_APP_COINBASE_PK || ''
```
[the remark this answers is not in the document]
Comment (cfluke-cb) [reply src/constants/index.ts:88]: done

File: src/pages/Beta/Buy/index.tsx (hunk 1 above)
Comment (cfluke-cb) [reply src/pages/Beta/Buy/index.tsx:64]: done

**Comment by cfluke-cb** [comment]
> hey @cfluke-cb thanks for the changes. I reviewed your screenshot with our designer. We wants to do something like this
> 
> submenu_example
> 
> When user hover on the Buy menu item, it will show 2 options. Let me know if I can help you in anyway

What is the expectation on mobile?

**Comment by SarjuHansaliya** [comment]
Here is how it should look in mobile

**Pull request closed without merging**